src/cow-react/modules/limitOrders/containers/LimitOrdersReceiptModal/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReceiptModal as ReceiptModalPure } from '@cow/modules/limitOrders/pure/ReceiptModal'
+import { useAtomValue } from 'jotai/utils'
+import { receiptAtom } from '@cow/modules/limitOrders/state/limitOrdersReceiptAtom'
+import { useCloseReceiptModal } from './hooks'
+import { useWeb3React } from '@web3-react/core'
+import { supportedChainId } from '@src/custom/utils/supportedChainId'
+import { CurrencyAmount, Fraction, Token } from '@uniswap/sdk-core'
+import JSBI from 'jsbi'
+
+function calculatePrice({
```

Comment:
nit: could you make a type with the params? this way the function definition is smaller, and we have some reusable type we can use.

If you do that, it wouldn't be longer needed, but when the function definition is too long, i prefer to extract the parameters in the function itself (instead of in the definition). The reason again is to make the definition, small and easy to read.